models/Redirect.php

```diff
@@ -176,9 +176,11 @@ protected static function makeValidator(
                 && $request->get('target_type') === self::TARGET_TYPE_STATIC_PAGE;
         });
 
-        $validator->sometimes('from_url', 'is_regex', static function (Fluent $request): bool {
-            return $request->get('match_type') === self::TYPE_REGEX;
-        });
+        if (method_exists($validator, 'validateIsRegex')) {
```

Comment:
Why is this required?

Reply:
Because the custom validator added in Plugin.php does not exist when running the tests somehow.

Reply:
I feel like we should probably fix that issue rather than having a hack in here 😜 Any ideas @jaxwilko?

Reply:
I agree but I spent the better part of the day trying and couldn't figure it out.

Reply:
I removed the "hack". If someone wants to take a shot at it, they're most welcome.

Reply:
Interesting, the custom is_regex validation in tests passes in github but fails in my local environment with this error:
```
$ ./artisan winter:test -p winter.redirect
Running tests for plugin: winter.redirect
PHPUnit 9.5.21 #StandWithUkraine

..........E

Time: 00:09.545, Memory: 46.50 MB

There was 1 error:

1) Winter\Redirect\Tests\Cases\RedirectManagerTest::testTargetCmsPageRedirectWithRegexSource
BadMethodCallException: Method Winter\Storm\Validation\Validator::validateIsRegex does not exist.

/home/w2c/w2c.studioazura.com/current/public/vendor/laravel/framework/src/Illuminate/Validation/Validator.php:1534
/home/w2c/w2c.studioazura.com/current/public/vendor/laravel/framework/src/Illuminate/Validation/Validator.php:609
/home/w2c/w2c.studioazura.com/current/public/vendor/laravel/framework/src/Illuminate/Validation/Validator.php:415
/home/w2c/w2c.studioazura.com/current/public/vendor/winter/storm/src/Database/Traits/Validation.php:308
/home/w2c/w2c.studioazura.com/current/public/vendor/winter/storm/src/Database/Traits/Validation.php:76
/home/w2c/w2c.studioazura.com/current/public/vendor/winter/storm/src/Support/Traits/Emitter.php:182
/home/w2c/w2c.studioazura.com/current/public/vendor/winter/storm/src/Database/Model.php:787
/home/w2c/w2c.studioazura.com/current/public/vendor/winter/storm/src/Database/Model.php:830
/home/w2c/w2c.studioazura.com/current/public/plugins/winter/redirect/tests/cases/RedirectManagerTest.php:455
```